packages/dynamite/dynamite/lib/src/builder/client.dart

```diff
@@ -430,17 +335,108 @@ return rawResponse.future;
+              } else if (acceptHeader.isNotEmpty) {
+                headersExpression = declareConst('_headers');
+              }
+
+              if (headersExpression != null) {
+                final headersCode = headersExpression
+                    .assign(
+                      literalMap(
+                        {if (acceptHeader.isNotEmpty) 'Accept': acceptHeader},
+                        refer('String'),
+                        refer('String'),
+                      ),
+                    )
+                    .statement
+                    .accept(state.emitter);
+
+                code.writeln(headersCode);
+              }
+
+              if (hasContentEncoding) {
+                code.writeln('Uint8List? _body;');
+              }
+              // Separate the declarations from the assignments
+              code.writeln();
+
+              if (hasAuthentication) {
+                buildAuthCheck(
+                  state,
+                  operation,
+                  spec,
+                  client,
+                  code,
+                );
+              }
+
+              for (final parameter in pathParameters.entries) {
+                code.writeln(buildParameterSerialization(parameter.value, parameter.key, state, allocate));
+              }
+              for (final parameter in headerParameters.entries) {
+                code.writeln(buildParameterSerialization(parameter.value, parameter.key, state, allocate));
+              }
+
+              if (bodyParameter != null) {
+                resolveMimeTypeEncode(bodyParameter.mimeType, bodyParameter.result, code);
+              }
+
+              buildUrlPath(pathEntry.key, parameters, code, state.emitter.allocator.allocate);
+
+              final statusCodes = responseEntry.value;
+              final returnValue = refer(
+                'DynamiteRawResponse<${bodyType.name}, ${headersType.name}>',
+                'package:dynamite_runtime/http_client.dart',
+              ).newInstance(const [], {
+                'response': refer(client).property('executeRequest').call([
+                  literalString(httpMethod),
+                  refer('_path'),
+                ], {
+                  if (acceptHeader.isNotEmpty || hasHeaderParameters || hasAuthentication || hasContentEncoding)
+                    'headers': refer('_headers'),
+                  if (operation.requestBody != null) 'body': refer('_body'),
+                  if (responses.values.isNotEmpty && !statusCodes.contains('default'))
+                    'validStatuses': literalConstSet(statusCodes),
+                }),
+                'bodyType': refer(bodyType.fullType ?? 'null'),
+                'headersType': refer(headersType.fullType ?? 'null'),
+                'serializers': refer(r'_$jsonSerializers'),
+              });
+
+              code.writeln(returnValue.returned.statement.accept(state.emitter));
+              return code.toString();
+            });
```

Comment:
Would it be possible to avoid changing the order of the import aliases this change introduces?

Reply:
We can try and play with the generation order but if not now it'll probably come in the following PR.
Let me rebase and check.

Reply:
I just rebased and can confirm that the order is going to stay